diff --git a/pandas/compat/__init__.py b/pandas/compat/__init__.py
index 60cfecd..c7abd3b 100644
--- a/pandas/compat/__init__.py
+++ b/pandas/compat/__init__.py
@@ -114,7 +114,7 @@ def _import_lzma():
             "Your installed Python is incomplete. "
             "Attempting to use lzma compression will result in a RuntimeError."
         )
-        warnings.warn(msg)
+        #warnings.warn(msg)
 
 
 def _get_lzma_file(lzma):
