diff --git a/scipy/stats/kde.py b/scipy/stats/kde.py
index ed082e3..7ec30a8 100755
--- a/scipy/stats/kde.py
+++ b/scipy/stats/kde.py
@@ -31,7 +31,10 @@ from numpy import (asarray, atleast_2d, reshape, zeros, newaxis, dot, exp, pi,
 import numpy as np

 # Local imports.
-from . import mvn
+try:
+    from . import mvn
+except ImportError:
+    mvn = None
 from ._stats import gaussian_kernel_estimate


diff --git a/scipy/stats/setup.py b/scipy/stats/setup.py
index 94dc721..abbb1f0 100755
--- a/scipy/stats/setup.py
+++ b/scipy/stats/setup.py
@@ -24,9 +24,9 @@ def configuration(parent_package='',top_path=None):
     )
 
     # add mvn module
-    config.add_extension('mvn',
-        sources=['mvn.pyf','mvndst.f'],
-    )
+    #config.add_extension('mvn',
+    #    sources=['mvn.pyf','mvndst.f'],
+    #)
 
     return config
 
