diff --git a/scipy/integrate/setup.py b/scipy/integrate/setup.py
index 11ce3d1..92b69f6 100755
--- a/scipy/integrate/setup.py
+++ b/scipy/integrate/setup.py
@@ -82,12 +82,12 @@ def configuration(parent_package='',top_path=None):
     ext._pre_build_hook = pre_build_hook

     # dop
-    ext = config.add_extension('_dop',
-                               sources=['dop.pyf'],
-                               libraries=['dop'],
-                               depends=dop_src,
-                               f2py_options=f2py_options)
-    ext._pre_build_hook = pre_build_hook
+    #ext = config.add_extension('_dop',
+    #                           sources=['dop.pyf'],
+    #                           libraries=['dop'],
+    #                           depends=dop_src,
+    #                           f2py_options=f2py_options)
+    #ext._pre_build_hook = pre_build_hook

     config.add_extension('_test_multivariate',
                          sources=quadpack_test_src)
diff --git a/scipy/optimize/__init__.py b/scipy/optimize/__init__.py
index 4ba647a..a9a0abd 100755
--- a/scipy/optimize/__init__.py
+++ b/scipy/optimize/__init__.py
@@ -392,7 +392,7 @@ from .minpack import *
 from .zeros import *
 from .lbfgsb import fmin_l_bfgs_b, LbfgsInvHessProduct
 from .tnc import fmin_tnc
-from .cobyla import fmin_cobyla
+#from .cobyla import fmin_cobyla
 from .nonlin import *
 from .slsqp import fmin_slsqp
 from ._nnls import nnls
diff --git a/scipy/optimize/_minimize.py b/scipy/optimize/_minimize.py
index f43b69f..a086a9d 100755
--- a/scipy/optimize/_minimize.py
+++ b/scipy/optimize/_minimize.py
@@ -29,7 +29,7 @@ from ._trustregion_constr import _minimize_trustregion_constr
 # constrained minimization
 from .lbfgsb import _minimize_lbfgsb
 from .tnc import _minimize_tnc
-from .cobyla import _minimize_cobyla
+#from .cobyla import _minimize_cobyla
 from .slsqp import _minimize_slsqp
 from ._constraints import (old_bound_to_new, new_bounds_to_old,
                            old_constraint_to_new, new_constraint_to_old,
diff --git a/scipy/optimize/setup.py b/scipy/optimize/setup.py
index f563cd5..5046920 100755
--- a/scipy/optimize/setup.py
+++ b/scipy/optimize/setup.py
@@ -76,10 +76,10 @@ def configuration(parent_package='', top_path=None):
                          depends=[join('tnc', 'tnc.h')],
                          **numpy_nodepr_api)

-    config.add_extension('_cobyla',
-                         sources=[join('cobyla', x) for x in [
-                             'cobyla.pyf', 'cobyla2.f', 'trstlp.f']],
-                         **numpy_nodepr_api)
+    #config.add_extension('_cobyla',
+    #                     sources=[join('cobyla', x) for x in [
+    #                         'cobyla.pyf', 'cobyla2.f', 'trstlp.f']],
+    #                     **numpy_nodepr_api)

     sources = ['minpack2.pyf', 'dcsrch.f', 'dcstep.f']
     config.add_extension('minpack2',
diff --git a/scipy/optimize/zeros.c b/scipy/optimize/zeros.c
index e8cb3dc..128ce67 100755
--- a/scipy/optimize/zeros.c
+++ b/scipy/optimize/zeros.c
@@ -201,6 +201,10 @@ PyObject *PyInit__zeros(void)
 {
     PyObject *m;

+    volatile long x;
+    x = (long)malloc(0);
+    free((void*)x);
+
     m = PyModule_Create(&moduledef);

     return m;
