diff --git a/scipy/special/__init__.py b/scipy/special/__init__.py
index 12a6ba6..ac99d0d 100755
--- a/scipy/special/__init__.py
+++ b/scipy/special/__init__.py
@@ -644,7 +644,6 @@ from .orthogonal import *
 from .spfun_stats import multigammaln
 from ._ellip_harm import (
     ellip_harm,
-    ellip_harm_2,
     ellip_normal
 )
 from ._lambertw import lambertw
diff --git a/scipy/special/_ellip_harm.py b/scipy/special/_ellip_harm.py
index f898f36..88b7955 100755
--- a/scipy/special/_ellip_harm.py
+++ b/scipy/special/_ellip_harm.py
@@ -1,7 +1,14 @@
 import numpy as np

 from ._ufuncs import _ellip_harm
-from ._ellip_harm_2 import _ellipsoid, _ellipsoid_norm
+
+
+def _raise_error(*cargs, **kwargs):
+    raise NotImplementedError
+
+
+_ellipsoid = _raise_error
+_ellipsoid_norm = _raise_error


 def ellip_harm(h2, k2, n, p, s, signm=1, signn=1):
diff --git a/scipy/special/setup.py b/scipy/special/setup.py
index 81eb09f..f93b31a 100755
--- a/scipy/special/setup.py
+++ b/scipy/special/setup.py
@@ -98,11 +98,11 @@ def configuration(parent_package='',top_path=None):
                          define_macros=define_macros,
                          extra_info=get_info("npymath"))

-    cfg = dict(get_system_info('lapack_opt'))
-    config.add_extension('_ellip_harm_2',
-                         sources=['_ellip_harm_2.c', 'sf_error.c',],
-                         **cfg
-                         )
+    #cfg = dict(get_system_info('lapack_opt'))
+    #config.add_extension('_ellip_harm_2',
+    #                     sources=['_ellip_harm_2.c', 'sf_error.c',],
+    #                     **cfg
+    #                     )

     # Cython API
     config.add_data_files('cython_special.pxd')
