commit 986ee79feb5f5324aa6cf07b1920db6d30a2fed7 (HEAD)
Author: Chigozie Nri <chigozie@gmail.com>
Date:   Mon Jul 13 20:59:34 2020 +0100

    Don't build openmp

diff --git a/setup.py b/setup.py
index 19331e4d0..06fb24830 100755
--- a/setup.py
+++ b/setup.py
@@ -66,6 +66,7 @@ def openmp_build_ext():
     class ConditionalOpenMP(build_ext):
 
         def can_compile_link(self):
+            return False
 
             cc = self.compiler
             fname = 'test.c'

diff --git a/skimage/restoration/_unwrap_2d.pyx b/skimage/restoration/_unwrap_2d.pyx
index ad0dfa2..a201541 100644
--- a/skimage/restoration/_unwrap_2d.pyx
+++ b/skimage/restoration/_unwrap_2d.pyx
@@ -3,7 +3,7 @@
 # cython: nonecheck=False
 # cython: wraparound=False
 
-cdef extern from *:
+cdef extern from "unwrap_2d_patch.h":
     void unwrap2D(double* wrapped_image,
                   double* unwrapped_image,
                   unsigned char* input_mask,
diff --git a/skimage/restoration/unwrap_2d_patch.h b/skimage/restoration/unwrap_2d_patch.h
new file mode 100644
index 0000000..eba3dd3
--- /dev/null
+++ b/skimage/restoration/unwrap_2d_patch.h
@@ -0,0 +1,13 @@
+//
+// Created by biolib on 9/2/20.
+//
+
+#ifndef SCIKIT_IMAGE_0_17_2_TEST_H
+#define SCIKIT_IMAGE_0_17_2_TEST_H
+
+#endif //SCIKIT_IMAGE_0_17_2_TEST_H
+
+void unwrap2D(double *wrapped_image, double *UnwrappedImage,
+              unsigned char *input_mask, int image_width, int image_height,
+              int wrap_around_x, int wrap_around_y,
+              char use_seed, unsigned int seed);

diff --git a/skimage/restoration/_unwrap_3d.pyx b/skimage/restoration/_unwrap_3d.pyx
index eac50e7..210d9a9 100644
--- a/skimage/restoration/_unwrap_3d.pyx
+++ b/skimage/restoration/_unwrap_3d.pyx
@@ -3,7 +3,7 @@
 # cython: nonecheck=False
 # cython: wraparound=False

-cdef extern from *:
+cdef extern from "unwrap_3d_patch.h":
     void unwrap3D(double* wrapped_volume,
                 double* unwrapped_volume,
                 unsigned char* input_mask,
