diff --git a/setup.py b/setup.py
index 6b6b644..af59dcf 100644
--- a/setup.py
+++ b/setup.py
@@ -211,6 +211,8 @@ class InstallLib(install_lib.install_lib):
         dst = os.path.join(self.install_dir, 'xgboost', 'lib', lib_name())
 
         global BUILD_TEMP_DIR   # pylint: disable=global-statement
+        if not BUILD_TEMP_DIR:
+            BUILD_TEMP_DIR = "../../../../cpython/build/3.7.4/host/lib/python3.7/site-packages"
         libxgboost_path = lib_name()
 
         dft_lib_dir = os.path.join(CURRENT_DIR, os.path.pardir, 'lib')
diff --git a/xgboost/dmlc-core/CMakeLists.txt b/xgboost/dmlc-core/CMakeLists.txt
index 141db72..dc73b4e 100644
--- a/xgboost/dmlc-core/CMakeLists.txt
+++ b/xgboost/dmlc-core/CMakeLists.txt
@@ -149,13 +149,13 @@ else()
 endif()
 
 # Check endianness
-include(TestBigEndian)
-test_big_endian(BIG_ENDIAN)
-if(BIG_ENDIAN)
-  set(DMLC_CMAKE_LITTLE_ENDIAN 0)
-else()
-  set(DMLC_CMAKE_LITTLE_ENDIAN 1)
-endif()
+#include(TestBigEndian)
+#test_big_endian(BIG_ENDIAN)
+#if(BIG_ENDIAN)
+#  set(DMLC_CMAKE_LITTLE_ENDIAN 0)
+#else()
+set(DMLC_CMAKE_LITTLE_ENDIAN 1)
+#endif()
 
 message(STATUS "${CMAKE_LOCAL}/build_config.h.in -> include/dmlc/build_config.h")
 configure_file("cmake/build_config.h.in" "include/dmlc/build_config.h")
